starbucks_dw/models/staging/stg_profile.sql

```diff
@@ -1 +1,22 @@
-/* TO BE IMPLEMENTED BY THE STUDENT */
+
+{{
+  config(
+    materialized = 'view',
+    )
+}}
+
+with
+    transformed_profile as (
+        select
+            id as customer_id,
+            COALESCE(gender, 'N/A') AS gender,
+            age,
+            COALESCE(income, 0) AS income,
+            TO_DATE(became_member_on::TEXT, 'YYYYMMDD') AS subscribed_date,
+            CURRENT_TIMESTAMP AS ingested_at
+        from {{ ref('profile') }}
+        where age >= 118
```

Comment:
The signal is the opposite. I think I confused you on that one when we talked during the class. But don't worry about it!